cluster/manifests/skipper/deployment.yaml

```diff
@@ -172,7 +172,7 @@ spec:
           - "-wait-for-healthcheck-interval={{ .Cluster.ConfigItems.skipper_wait_for_healthcheck_interval }}"
 {{ if ne .Cluster.ConfigItems.skipper_routesrv_enabled "exec" }}
           - "-kubernetes-east-west-range-domains=ingress.cluster.local"
-          - "-kubernetes-east-west-range-predicates=ClientIP(\"10.2.0.0/15\", \"{{ .Values.vpc_ipv4_cidr }}\")"
+          - "-kubernetes-east-west-range-predicates=ClientIP({{ if eq .Cluster.Provider "zalando-eks" }}{{if eq .Cluster.ConfigItems.eks_ip_family "ipv4"}}\"{{ .Values.vpc_ipv4_cidr }}\"{{else}}\"{{join .Values.vpc_ipv6_cidrs `\",\"`}}\"{{end}}{{else}}\"10.2.0.0/15\", \"{{ .Values.vpc_ipv4_cidr }}\"{{end}})"
```

Comment:
Would it be possible to define `cluster_internal_cidrs` variable to use it like:
```yaml
- "-kubernetes-east-west-range-predicates=ClientIP(\"{{ join $cluster_internal_cidrs `\",\"` }}\")"
...
- "-forwarded-headers-exclude-cidrs={{ join $cluster_internal_cidrs "," }}"
```

Reply:
Good idea, will do!

Reply:
Updated. This also needs a few changes in CLM: https://github.com/zalando-incubator/cluster-lifecycle-manager/pull/825